icons/pr-dot-co.svg

```diff
@@ -1 +1 @@
-<svg role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>pr.co icon</title><path d="M11.9998 4.67c1.876 0 3.7518.7157 5.1832 2.1468l.0167.0167.0053.0055.0014.0015.0016.0014c.9304.9393 1.5546 2.0673 1.873 3.2602l1.7906-1.7905c.7157-.7157 1.8761-.7157 2.5916 0 .7157.7155.7157 1.8758 0 2.5913l-6.2802 6.2803c-1.4314 1.4314-3.3073 2.1468-5.1832 2.1468-1.8758 0-3.7517-.7154-5.1831-2.1468-.9442-.9442-1.5768-2.0816-1.898-3.285L3.128 15.6887c-.7154.716-1.8758.716-2.5915 0-.7153-.7155-.7153-1.8758 0-2.5915l6.2478-6.2477a4.8688 4.8688 0 01.0324-.0326h.0002-.0002.0002C8.248 5.3857 10.1239 4.67 11.9999 4.67zm2.5918 9.9216c1.431-1.4312 1.431-3.752 0-5.1832-1.4314-1.4312-3.7521-1.4312-5.1833 0v.0002c-1.4314 1.431-1.4314 3.752 0 5.1832 1.4312 1.4312 3.7517 1.431 5.1831 0h.0002Z"/></svg>
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>pr.co icon</title><path d="M12 4.67c-1.8758 0-3.7517.7158-5.183 2.147h-.0003c-.0108.0108-.0217.0214-.0323.0322l-6.248 6.248c-.7152.7157-.7152 1.8758 0 2.5913.7158.7159 1.8763.7159 2.5918 0l1.7903-1.7904c.3213 1.2034.954 2.3407 1.8982 3.2849C8.248 18.6144 10.1242 19.33 12 19.33c1.876 0 3.7516-.7156 5.183-2.147l6.2802-6.2802c.7157-.7155.7157-1.8758 0-2.5913-.7155-.7157-1.876-.7157-2.5917 0l-1.7903 1.7908c-.3185-1.193-.9428-2.321-1.8731-3.2602l-.0016-.0016-.0016-.0016-.0052-.0052-.0167-.0167C15.7516 5.3858 13.876 4.67 12 4.67zm0 3.665c.938 0 1.876.3577 2.5917 1.0733 1.431 1.4312 1.431 3.7522 0 5.1834h-.0004c-1.4314 1.431-3.7518 1.4312-5.183 0-1.4313-1.4312-1.4313-3.752 0-5.183v-.0004C10.124 8.6927 11.062 8.335 12 8.335z"/></svg>
```

Comment:
LGTM - though the PR (red) has two points toward the top left that could potentially be reduced to one. Any chance you could take a quick look?

![image](https://user-images.githubusercontent.com/1830867/103992846-a1fe4900-518c-11eb-87f9-34b9b0430299.png)